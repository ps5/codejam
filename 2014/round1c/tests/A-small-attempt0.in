100
1/64
130/599
3/8
1/16
3/128
29/32
15/128
59/256
16/219
1/8
1/4
1/64
89/128
3/32
3/16
3/16
3/4
1/256
1/32
326/907
1/4
3/16
23/64
1/64
85/128
13/64
601/827
1/64
1/4
11/64
5/32
11/16
109/256
267/470
7/256
29/512
45/128
441/494
29/128
1/128
381/397
13/16
1/2
1/64
65/256
1/256
1/32
1/64
7/16
47/64
15/256
7/16
1/64
1/64
1/128
315/512
1/20
133/187
3/64
3/512
3/256
1/32
27/128
29/128
5/64
1/64
733/881
27/32
1/8
9/256
397/920
73/185
1/8
5/83
5/256
77/334
1/2
583/797
1/2
81/256
3/256
9/466
301/489
1/16
1/2
1/32
39/128
279/662
31/256
11/32
1/16
35/256
5/8
1/4
1/2
7/128
39/128
9/128
221/232
1/8
