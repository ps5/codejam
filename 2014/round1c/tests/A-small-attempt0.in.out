Case #1: 6
Case #2: impossible
Case #3: 2
Case #4: 4
Case #5: 6
Case #6: 1
Case #7: 4
Case #8: 3
Case #9: impossible
Case #10: 3
Case #11: 2
Case #12: 6
Case #13: 1
Case #14: 4
Case #15: 3
Case #16: 3
Case #17: 1
Case #18: 8
Case #19: 5
Case #20: impossible
Case #21: 2
Case #22: 3
Case #23: 2
Case #24: 6
Case #25: 1
Case #26: 3
Case #27: impossible
Case #28: 6
Case #29: 2
Case #30: 3
Case #31: 3
Case #32: 1
Case #33: 2
Case #34: impossible
Case #35: 6
Case #36: 5
Case #37: 2
Case #38: impossible
Case #39: 3
Case #40: 7
Case #41: impossible
Case #42: 1
Case #43: 1
Case #44: 6
Case #45: 2
Case #46: 8
Case #47: 5
Case #48: 6
Case #49: 2
Case #50: 1
Case #51: 5
Case #52: 2
Case #53: 6
Case #54: 6
Case #55: 7
Case #56: 1
Case #57: impossible
Case #58: impossible
Case #59: 5
Case #60: 8
Case #61: 7
Case #62: 5
Case #63: 3
Case #64: 3
Case #65: 4
Case #66: 6
Case #67: impossible
Case #68: 1
Case #69: 3
Case #70: 5
Case #71: impossible
Case #72: impossible
Case #73: 3
Case #74: impossible
Case #75: 6
Case #76: impossible
Case #77: 1
Case #78: impossible
Case #79: 1
Case #80: 2
Case #81: 7
Case #82: impossible
Case #83: impossible
Case #84: 4
Case #85: 1
Case #86: 5
Case #87: 2
Case #88: impossible
Case #89: 4
Case #90: 2
Case #91: 4
Case #92: 3
Case #93: 1
Case #94: 2
Case #95: 1
Case #96: 5
Case #97: 2
Case #98: 4
Case #99: impossible
Case #100: 3
